### PR TITLE
Individuals input restrictions

### LINKED ISSUE #79: Sample sets: The page should not be static
If things are changed in the individuals page, like adding a new sample set, then this should be reflected in the sample sets page

### LINKED ISSUE #93: Individuals: Select sample sets is not updating
The multiple choice alternatives in the "Select sample sets"  box are not updating if one changes the sample sets to a new previously not existing ID. This is because it uses the sample sets table to generate the options, and updates of sample sets IDs on the individuals page are not sent to the sample sets table.

### PR DESCRIPTION
**Pull request by almanilsson**
This PR makes `IndividualsTable` dependent on `SampleSetsTable` (thus moving the order of the classes in the `datastore` script, and the order of which they are created in `main` `app` and `model`), in order to be able to get **updated sample sets** to appear as choices **in the selection box and the batch reassignment boxes.** 

This PR thus changes the **selection box options**, so they are **updated when a a new sample set is added** in the sample sets page, which closes #93. 

Moreover the the **batch reassignment boxes** have been **made into selection boxes** instead of Intinput, because the **user should not be able to add things to a nonexistent sample set**,. It thus closes #79. These also now contain **automatically updated options.** 

The changes can be seen in the _Individuals_ page sidebar

image

Finally, this PR changes the `SampleSet` class variable "id" to "smaple_set_id",  since in python "id" should not be used as a variable name since it shadows the built in id() function.

### CODE REVIEW

File: src/tseda/datastore.py [hunk 1]
```diff
@@ -36,15 +38,186 @@ def preprocess(tsm):
...
+            formatters=self.formatters,
+            editors=self.editors,
+        )
+        return pn.Column(
+            pn.pane.Markdown("### Sample Set Table"), self.tooltip, table
+        )
+
+    def get_ids(self):
+        if isinstance(self.table, pd.DataFrame):
+            return [i for i in range(len(self.table["name"].tolist()))]
```
Comment (almanilsson) [inline src/tseda/datastore.py:137]: This solution is not ideal, but since the table does not actually contain the indices this was a quick fix

Comment (percyfal) [reply src/tseda/datastore.py:137]: The index values should be available through `df.index.values` or something like that I think.

**Review by percyfal — APPROVED** [review]
LGTM!

File: src/tseda/datastore.py (hunk 1 above)
Comment (almanilsson) [reply src/tseda/datastore.py:137]: Ah thank you!

**Pull request merged**